frank/utilities.py

```diff
@@ -185,6 +186,9 @@ def normalize_uv(u, v, wle):
         u and v coordinates normalized by observing wavelength
     """
 
+    logging.info('  Normalizing u and v by observing wavelength of'
+                 ' {} m'.format(wle))
+
```

Comment:
I genuinely think this is unnecessary. If you *really* want to log this function, do it at the debug level, not info

Reply:
I don't see this as unnecessary but as a useful status report to the user. I'd prefer to leave at the info level.

Reply:
Moved to fit.py